pyproject.toml

```diff
@@ -21,7 +21,7 @@ classifiers = [
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
 dependencies = [
-    "pyglet>=2.0.10,<2.1",
+    "pyglet==2.0.11",
```

Comment:
Should we continue to auto-pull newer versions?

`"pyglet>=2.0.11,<2.1",`

Reply:
Reverted, we don't use protected anymore per the suggestions caffienepills made anymore.